content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -0,0 +1,64 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Ephemeral Volumes
+content_type: concept
+weight: 20
```

Comment:
I need to check the weight....

Reply:
... and remove this file! It's supposed to be merged via https://github.com/kubernetes/website/pull/22438